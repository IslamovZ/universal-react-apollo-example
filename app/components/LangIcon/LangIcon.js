import React, { PropTypes } from 'react';

const LangIcon = ({ language }) => (
  <span className={`flag-icon flag-icon-${language}`} />
);

LangIcon.propTypes = {
  language: PropTypes.string.isRequired,
};

export default LangIcon;
