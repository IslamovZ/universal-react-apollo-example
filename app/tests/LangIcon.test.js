import React from 'react';
import renderer from 'react-test-renderer';

import LangIcon from 'components/LangIcon/LangIcon';

describe('LangIcon Component', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <LangIcon />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
