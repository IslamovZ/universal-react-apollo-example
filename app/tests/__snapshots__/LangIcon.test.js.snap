exports[`LangIcon Component renders correctly 1`] = `
<span
  className="flag-icon flag-icon-undefined" />
`;
